src/Commands/DrupalCommands.php

```diff
@@ -78,45 +78,47 @@ public function setRuntimeConfig(ConsoleCommandEvent $event)
      */
     public function siteInstall(array $options = [
-      'root'              => InputOption::VALUE_REQUIRED,
-      'base-url'          => InputOption::VALUE_REQUIRED,
-      'site-name'         => InputOption::VALUE_REQUIRED,
-      'site-mail'         => InputOption::VALUE_REQUIRED,
-      'site-profile'      => InputOption::VALUE_REQUIRED,
-      'site-update'       => InputOption::VALUE_REQUIRED,
-      'site-locale'       => InputOption::VALUE_REQUIRED,
-      'account-name'      => InputOption::VALUE_REQUIRED,
-      'account-password'  => InputOption::VALUE_REQUIRED,
-      'account-mail'      => InputOption::VALUE_REQUIRED,
-      'database-user'     => InputOption::VALUE_REQUIRED,
-      'database-password' => InputOption::VALUE_REQUIRED,
-      'database-host'     => InputOption::VALUE_REQUIRED,
-      'database-port'     => InputOption::VALUE_REQUIRED,
-      'database-name'     => InputOption::VALUE_REQUIRED,
-      'sites-subdir'      => InputOption::VALUE_REQUIRED,
-    ])
+            'root' => InputOption::VALUE_REQUIRED,
+            'base-url' => InputOption::VALUE_REQUIRED,
+            'site-name' => InputOption::VALUE_REQUIRED,
+            'site-mail' => InputOption::VALUE_REQUIRED,
+            'site-profile' => InputOption::VALUE_REQUIRED,
+            'site-update' => InputOption::VALUE_REQUIRED,
+            'site-locale' => InputOption::VALUE_REQUIRED,
+            'account-name' => InputOption::VALUE_REQUIRED,
+            'account-password' => InputOption::VALUE_REQUIRED,
+            'account-mail' => InputOption::VALUE_REQUIRED,
+            'database-type' => InputOption::VALUE_REQUIRED,
+            'database-user' => InputOption::VALUE_OPTIONAL,
+            'database-password' => InputOption::VALUE_OPTIONAL,
+            'database-host' => InputOption::VALUE_OPTIONAL,
+            'database-port' => InputOption::VALUE_OPTIONAL,
+            'database-name' => InputOption::VALUE_OPTIONAL,
+            'sites-subdir' => InputOption::VALUE_REQUIRED,
+        ])
     {
         $drush = $this->getConfig()->get('runner.bin_dir').'/drush';
-        $task = $this->taskDrush($drush)
-          ->root($options['root'])
-          ->siteName($options['site-name'])
-          ->siteMail($options['site-mail'])
-          ->locale($options['site-locale'])
-          ->accountMail($options['account-mail'])
-          ->accountName($options['account-name'])
-          ->accountPassword($options['account-password'])
-          ->databaseUser($options['database-user'])
-          ->databasePassword($options['database-password'])
-          ->databaseHost($options['database-host'])
-          ->databasePort($options['database-port'])
-          ->databaseName($options['database-name'])
-          ->sitesSubdir($options['sites-subdir'])
-          ->siteProfile($options['site-profile']);
+        $task  = $this->taskDrush($drush)
+                     ->root($options['root'])
```

Comment:
This is also very strangely indented. Let's first of all not align the assignment operators because this is brittle, and let's not overindent the chained method calls.